quartz/components/styles/explorer.scss

```diff
@@ -1,14 +1,69 @@
 @use "../../styles/variables.scss" as *;
 
+@media all and ($mobile) {
+  .page > #quartz-body {
+    // Shift page position when toggling Explorer on mobile.
+    & > :not(.sidebar.left:has(.explorer)) {
+      transform: translateX(0);
+      transition: transform 300ms ease-in-out;
+    }
+    &.lock-scroll > :not(.sidebar.left:has(.explorer)) {
+      transform: translateX(100dvw);
+      transition: transform 300ms ease-in-out;
+    }
+
+    // Sticky top bar (stays in place when scrolling down on mobile).
+    .sidebar.left:has(.explorer) {
+      box-sizing: border-box;
+      position: sticky;
+      background-color: var(--light);
+    }
+
+    // Hide Explorer on mobile until done loading.
+    // Prevents ugly animation on page load.
+    .hide-until-loaded ~ #explorer-content {
+      display: none;
+    }
+  }
+}
+
 .explorer {
   display: flex;
```

Comment:
Might be better showing explorer as `inline-block`.

As reported, flex shrinks if there isn't enough space. So if the title bar becomes squeezy the explorer icon will become smaller, making it harder to tap on and also impacts `explorer-content` alignment, as demoed below:

## Before

<img width="437" alt="Screenshot 2024-11-02 at 2 26 46 AM" src="https://github.com/user-attachments/assets/b07eb367-c25a-47a3-bd55-d5a4a1b1e603">

## After

<img width="437" alt="Screenshot 2024-11-02 at 2 36 51 AM" src="https://github.com/user-attachments/assets/89a00cef-3302-4c5c-add5-10fa61679f40">

Reply:
Can you verify this is not related to your use of a customized grid layout for your left sidebar over Quartz's default flex for the left sidebar?

Reply:
I didn't alter the flex layout in mobile mode.

<img width="502" alt="Screenshot 2024-11-02 at 3 00 05 AM" src="https://github.com/user-attachments/assets/4b05a741-41fa-484f-86da-ea7efa795d19">


I think this is easily reproducible if you make the title bar more squeezy by having a longer page title or use "responsive" dimension in Chromium browsers and make the page narrower. It's really related to the `explorer` container itself being a `flex` container causing its containing component (the hamburger button) to shrink

<img width="453" alt="Screenshot 2024-11-02 at 3 01 37 AM" src="https://github.com/user-attachments/assets/372c24db-2680-411f-81f3-15c89b331cb2">

Reply:
Thanks. I'll look into it.

I asked because your previous suggested change didn't work in default Quartz.

I'll let you know my conclusion soon.

Once again, thanks for the reports!